app/controllers/alchemy/base_controller.rb

```diff
@@ -32,7 +32,8 @@ def not_found_error!(msg = "Not found \"#{request.fullpath}\"")
 
     # Shortcut for Alchemy::I18n.translate method
     def _t(key, *args)
-      I18n.t(key, *args)
+      ActiveSupport::Deprecation.warn("Alchemys `_t` method is deprecated! Use `Alchemy.t` instead.", caller.unshift)
```

Comment:
What does `caller.unshift` do here? I've only seen it used with just plain `caller`. Just out of interest :)